.github/workflows/update-rock.yaml

```diff
@@ -32,13 +32,17 @@ jobs:
     steps:
       - name: Install dependencies
         run: |
+          sudo snap install jq
           sudo snap install yq
-
       - id: latest-release
-        uses: pozetroninc/github-action-get-latest-release@v0.6.0
-        with:
-          repository: ${{ inputs.source-repo }}
-          excludes: prerelease, draft
+        run: |
```

Comment:
Add `name:` for documentation? Can't tell what this step does.

Reply:
Fixed!